src/views/profile/__tests__/ProfileDetail.spec.ts

```diff
@@ -0,0 +1,146 @@
+import ProfileDetail from "@/views/profile/ProfileDetail.vue";
+import { render, waitFor } from "@testing-library/vue";
+import { isAddress } from "ethers/lib/utils";
```

Comment:
Let's use `web3` instead of `ethers`

Reply:
I added `isAddress` to the useWeb3 functions